spring-security-sso/spring-security-sso-kerberos/src/main/java/kerberos/kdc/MiniKdcConfigBuilder.java

```diff
@@ -0,0 +1,65 @@
+package kerberos.kdc;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Paths;
+import java.util.Objects;
+
+class MiniKdcConfigBuilder {
+
+	private static final int FIELDS_COUNT = 3;
+	private String workDir;
+	private String confDir;
+	private String keytabName;
+	private String[] principals = { "principal/localhost" };
```

Comment:
Why not a List? wouldn't it be easier to handle?

Reply:
refactored to list usage, thank you